projects/Amlogic/linux/linux.aarch64.conf

```diff
@@ -3111,32 +3109,32 @@ CONFIG_VIDEO_CX25840=m
 CONFIG_CXD2880_SPI_DRV=m
 # end of Media SPI Adapters
 
-CONFIG_MEDIA_TUNER=m
+CONFIG_MEDIA_TUNER=y
```

Comment:
any reason for this ?
every other project builds the dvb drivers as modules

Reply:
That's result of `make oldconfig` so there must be something which is set to `=y` and thus whole chain becomes `=y` too.

Reply:
Yep, I just accepted the defaults which is what will be used during the build (assuming the build doesn't halt completely waiting for an answer to a prompt, and is quite likely). If these extra options need fine tuning then I would defer that to someone more intimately involved with the Amlogic project.

BTW Allwinner (both `arm` and `aarch64` configs) specifies `CONFIG_MEDIA_TUNER=y`, while all other projects (including Amlogic) use `CONFIG_MEDIA_TUNER=m` (which is now being set to `=y` in this PR). I'm not sure what is forcing this option to `=y`, or if it should be `=m`.

Reply:
Further to my previous comment `=y` is the default for `CONFIG_MEDIA_TUNER`, and setting it to `=m` has no effect (it remains `=y`), so something else is setting it `=y`...